1001-2000/LIT1942Mashaf.xml

```diff
@@ -35,7 +35,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
-         <langUsage><language ident="en">English</language><language ident="gez">Gǝʿǝz</language></langUsage>
+         <abstract>
+            <p>Collection of medical prescriptions in a mixture of Gǝʾǝz and Amharic, which is reported to originate from Jerusalem. It differs
+               from <ref type="work" corresp="LIT7529Madhanit"/> and <ref type="work" corresp="LIT7530MashafaMad"/>.</p>
```

Comment:
If this record refers currently only to the version attested in BLorient828, I think it would make sense to mention this in the abstracz